src/wallet/init.cpp

```diff
@@ -82,6 +82,12 @@ void WalletInit::AddWalletOptions(ArgsManager& argsman) const
     argsman.AddHiddenArgs({"-dblogsize", "-flushwallet", "-privdb"});
 #endif
 
+#ifdef USE_SQLITE
+    argsman.AddArg("-unsafesqlitesync", "Set SQLite synchronous=OFF to disable waiting for the database to sync to disk. This is unsafe and can cause data loss and corruption. This option is only used by tests to improve their performance (default: false)", ArgsManager::ALLOW_ANY | ArgsManager::DEBUG_ONLY, OptionsCategory::WALLET_DEBUG_TEST);
```

Comment:
Since this is a boolean argument, why not `s/ArgsManager::ALLOW_ANY/ArgsManager::ALLOW_BOOL/`

Reply:
Done